modules/security-context-constraints-psa-coexistence.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-and-managing-pod-security-admission.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="security-context-constraints-psa-coexistence_{context}"]
+= Understanding pod security admission coexistence
+
+The Kubernetes pod security admission controller and the security context constraints controller work together to enforce both policies simultaneously, while preserving backward compatibility as follows:
```

Comment:
```suggestion
The Kubernetes pod security admission controller enforces pod security profiles. The security context constraints controller enforces security context constraints (SCC). The controllers work together using the following webhooks to enforce security policies and preserve backward compatibility:
```
This sentence is pretty long. I suggest introducing the controllers with two shorter sentences, then introducing the webhooks as the means through which the controllers work together.